src/get_webhook_data.py

```
@@ -66,6 +67,16 @@ class InputRequest(BaseModel):
 ## Variables
 
 token = os.environ.get("GIT_TOKEN")
+if not token:
+    try:
+        secret_id = "Daily_Range_GITHUB_TOKEN"
+        project_id = "367674009031"
+        token = access_secret_version(project_id, secret_id)
```

Ruff (0.8.2)

`access_secret_version` may be undefined, or defined from star imports

(F405)